evaluate_tflite.py

```diff
@@ -98,11 +98,14 @@ def main(args, _):
         predictions.append(msg['prediction'])
         wavlist.append(msg['wav'])
 
+    # Print test summary
     wer, cer, samples = calculate_report(wav_filenames, ground_truths, predictions, losses)
     mean_loss = np.mean(losses)
+    print('Test - WER: %f, CER: %f, loss: %f' % (wer, cer, mean_loss))
+    print('-' * 80)
 
-    print('Test - WER: %f, CER: %f, loss: %f' %
-          (wer, cer, mean_loss))
+    # Print some examples
+    print_report(samples)
```

Comment:
Wouldn't it make sense to factorize a bit more and also include the `print()` statement above into `print_report` ?

Reply:
I thought about this but they are different. evaluate.py also prints the datset path, evaluate_tflite.py doesnt.

Reply:
I don't think it would be a big deal ; it's likely a side-effect that in the past other code was behaving like `evaluate_tflite.py` and over time because we kept same duplicated code, they diverged.

Reply:
So move this too?
And leave out the dataset name or make it an optional param?

Reply:
> So move this too?
I'd say yes

> And leave out the dataset name or make it an optional param?
Sorry, I'm confused to what you refer here.

Reply:
tflite: print('Test - WER: %f, CER: %f, loss: %f' % (wer, cer, mean_loss))
eval: print('Test on %s - WER: %f, CER: %f, loss: %f' % (dataset, wer, cer, mean_loss))

Reply:
Just include the dataset. There's no good reason not to show it.

Reply:
there is no dataset in tflite. only a csv file the user sets as argument. i think that was the reason not to show it.
But we can use the csv path instead of the dataset name.

Reply:
> there is no dataset in tflite. only a csv file the user sets as argument. i think that was the reason not to show it.
> But we can use the csv path instead of the dataset name.

fine by me :)